docs/r2/get_started.ipynb

```diff
@@ -87,16 +87,33 @@
       "metadata": {
         "id": "6B95Hb6YVgPZ",
         "colab_type": "code",
-        "colab": {}
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 85
+        },
+        "outputId": "9f27f106-41e8-40bd-f73d-6abbb1314643"
       },
       "cell_type": "code",
       "source": [
         "!pip install -q tf-nightly-2.0-preview\n",
+        "\n",
         "# Load the TensorBoard notebook extension\n",
         "%load_ext tensorboard.notebook "
       ],
-      "execution_count": 0,
-      "outputs": []
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "\u001b[K    100% |████████████████████████████████| 79.1MB 325kB/s \n",
```

Comment:
Is standard practice for our docs to include these kinds of progress
bars in the notebook output? It seems like it could appear misleading:
if you load the notebook into a fresh Colab VM, you’ll see “100%
downloaded”, “trained 5 epochs”, etc. even though nothing has actually
happened.

Reply:
See note above.
No, we usually do not like to save notebooks with outputs. Usually, we execute the notebook during import with docs_to_devsite for display on tensorflow.org.
But we need to do it here because `%tensorboard` is not supported in the TF docker images and we need to add additional functionality to docs_to_devsite to handle this. See: b/124892469 and b/124898762

Reply:
Not sure if this is a good data point but I 👍 based on observations from other notebooks like this:
https://colab.research.google.com/github/aymericdamien/TensorFlow-Examples/blob/master/notebooks/3_NeuralNetworks/convolutional_network.ipynb

Is it weird to show such executed outputs in verbatim?

Reply:
TF Probability prefers to save their notebooks with outputs: https://github.com/tensorflow/probability/blob/master/tensorflow_probability/examples/jupyter_notebooks/Probabilistic_PCA.ipynb

But, generally, we don't like saving notebooks with outputs because it makes the diffs more difficult and it bloats the repo with generated images.

But, in your case, we __must__ to do it here because Colab seems to be the only place that renders your notebooks and we can use that as part of our import script.

Reply:
I should note: If you want to selectively delete cells and then save with outputs, that's perfectly fine. That's what I'm doing with the %tensorboard output cell.
You are welcome to remove the `pip install` output

Reply:
Great—thanks for the explanation.